### DISCUSSION TITLE
"Sonar" stands for what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#241] [Galaxy Trivia] [answered]
**Hint:** 🔉🧭🏆

**Comment by thuongtruong1009** [comment, marked as answer]
"Sonar" is an acronym that stands for "Sound Navigation And Ranging". It is a technique that uses sound waves to detect and locate objects underwater, and is commonly used for various purposes such as military, commercial, and scientific applications. Sonar works by emitting sound waves and then detecting the echoes that bounce back off of objects in the water, which can provide information about the location, size, and shape of those objects.
